chat-plugins/report.js

```diff
@@ -50,80 +52,77 @@ exports.commands = {
 
 	reports: function (target, room, user, connection, cmd) {
 		if (!user.can('pban')) return this.errorReply('/reports - Access denied.');
-		if (!target) var target = '';
+		if (!target) target = '';
 		target = target.trim();
 
-		var cmdParts = target.split(' ');
-		var cmd = cmdParts.shift().trim().toLowerCase();
-		var params = cmdParts.join(' ').split(',').map(function (param) { return param.trim(); });
+		let cmdParts = target.split(' ');
+		cmd = cmdParts.shift().trim().toLowerCase();
+		let params = cmdParts.join(' ').split(',').map(param => param.trim());
```

Comment:
shouldn't this be:
``let params = cmdParts.join(' ').split(',').map(param => { param.trim() });``